data_model/src/lib.rs

```diff
@@ -1843,55 +1859,13 @@ pub fn current_time() -> core::time::Duration {
         .expect("Failed to get the current system time")
 }
 
-#[cfg(feature = "http")]
-pub mod http {
-    //! Structures related to HTTP communication
-
-    use iroha_data_model_derive::model;
-    use iroha_schema::IntoSchema;
-    use iroha_version::declare_versioned_with_scale;
-
-    pub use self::model::*;
-    use crate::prelude::QueryOutput;
-
-    declare_versioned_with_scale!(BatchedResponse<T> 1..2, Debug, Clone, iroha_macro::FromVariant, IntoSchema);
-
-    #[model]
-    pub mod model {
-        use getset::Getters;
-        use iroha_version::version_with_scale;
-        use parity_scale_codec::{Decode, Encode};
-        use serde::{Deserialize, Serialize};
-
-        use super::*;
-
-        /// Batched response of a query sent to torii
-        #[derive(Debug, Clone, Getters, Decode, Encode, Deserialize, Serialize, IntoSchema)]
-        #[version_with_scale(version = 1, versioned_alias = "BatchedResponse")]
-        #[getset(get = "pub")]
-        #[must_use]
-        pub struct BatchedResponseV1<T> {
-            /// Current batch
-            pub batch: T,
-            /// Index of the next element in the result set. Client will use this value
-            /// in the next request to continue fetching results of the original query
-            pub cursor: crate::query::cursor::ForwardCursor,
-        }
-    }
-
-    impl From<BatchedResponseV1<Self>> for QueryOutput {
-        #[inline]
-        fn from(source: BatchedResponseV1<Self>) -> Self {
-            source.batch
-        }
-    }
+declare_versioned_with_scale!(BatchedResponse<T> 1..2, Debug, Clone, iroha_macro::FromVariant, IntoSchema);
```

Comment:
unlike other structs/enums, now this is not in the `model` module. Is that ok for API?

Reply:
No, looks fine to me, I used it in the same way we do for other versioned structs